src/commands/ring/create.ts

```diff
@@ -82,8 +82,16 @@ export const execute = async (
     logger.info(`Successfully created ring: ${ringName} for this project!`);
     await exitFn(0);
   } catch (err) {
-    logger.error(`Error occurred while creating ring: ${ringName}`);
-    logger.error(err);
+    logError(
+      buildError(
+        errorStatusCode.CMD_EXE_ERR,
+        {
+          errorKey: "ring-err-create",
```

Comment:
`ring-err-create` -> `ring-create-cmd-failed`. like we already have `hld-init-cmd-failed`, `infra-scaffold-cmd-failed` etc.